packages/stems/src/components/SelectablePill/SelectablePill.module.css

```diff
@@ -0,0 +1,67 @@
+.pill {
+  cursor: pointer;
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  color: var(--text-subdued);
+  user-select: none;
+  height: 24px;
```

Comment:
Might be out of the loop, but likely these should also be using grid units?

Reply:
yeah I thought about this, I wasn't sure if units should be used for things other than spacing (i.e. gaps, margins, padding) etc

Reply:
@Kyle-Shanks @dylanjeffers @sliptype @amendelsohn what are your thoughts on using grid units for things like `height` and `width`?

Reply:
Ideally everything conforms to our grid so I say let's use grid units here too!